amgut/db/patches/0036.sql

```diff
@@ -0,0 +1,389 @@
+INSERT INTO survey_question_response_type (survey_question_id, survey_response_type) VALUES (174, 'SINGLE');
+INSERT INTO survey_question_response_type (survey_question_id, survey_response_type) VALUES (175, 'SINGLE');
+INSERT INTO survey_question_response_type (survey_question_id, survey_response_type) VALUES (176, 'SINGLE');
+INSERT INTO survey_question_response_type (survey_question_id, survey_response_type) VALUES (177, 'SINGLE');
+INSERT INTO survey_question_response_type (survey_question_id, survey_response_type) VALUES (178, 'SINGLE');
+INSERT INTO survey_question_response_type (survey_question_id, survey_response_type) VALUES (179, 'SINGLE');
+INSERT INTO survey_question_response_type (survey_question_id, survey_response_type) VALUES (180, 'SINGLE');
+INSERT INTO survey_question_response_type (survey_question_id, survey_response_type) VALUES (181, 'SINGLE');
+INSERT INTO survey_question_response_type (survey_question_id, survey_response_type) VALUES (182, 'SINGLE');
+INSERT INTO survey_question_response_type (survey_question_id, survey_response_type) VALUES (183, 'SINGLE');
+INSERT INTO survey_question_response_type (survey_question_id, survey_response_type) VALUES (184, 'SINGLE');
+
+----------------------------------------------------------
+-- survey_response
+----------------------------------------------------------
+INSERT INTO survey_response (american, british) VALUES
+('I have not increased my intake', 'I have not increased my intake'),
+('Kimchi', 'Kimchi'),
+('Sauerkraut', 'Sauerkraut'),
+('Fermented beans/Miso/Natto', 'Fermented beans/Miso/Natto'),
+('Pickled vegetables', 'Pickled vegetables'),
+('Tempeh', 'Tempeh'),
+('Fermented tofu', 'Fermented tofu'),
+('Kefir (water)', 'Kefir (water)'),
+('Kefir (milk)', 'Kefir (milk)'),
+('Cottage cheese', 'Cottage cheese'),
+('Yogurt/lassi', 'Yogurt/lassi'),
+('Sour cream/creme fraiche', 'Sour cream/creme fraiche'),
+('Fermented fish', 'Fermented fish'),
+('Fish sauce', 'Fish sauce'),
+('Fermented bread/sourdough/injera', 'Fermented bread/sourdough/injera'),
+('Kombucha', 'Kombucha'),
+('Chicha', 'Chicha'),
+('Beer', 'Beer'),
+('Cider', 'Cider'),
+('Wine', 'Wine'),
+('Mead', 'Mead');
+
+INSERT INTO survey_response (american, british) VALUES
+('Point Loma/Ocean Beach, San Diego, California, USA', 'Point Loma/Ocean Beach, San Diego, California, USA'),
+('La Jolla, San Diego, California, USA', 'La Jolla, San Diego, California, USA'),
+('Encinitas, California, USA', 'Encinitas, California, USA'),
+('Southern California, USA', 'Southern California, USA'),
+('Central California, USA', 'Central California, USA'),
+('Northern, California', 'Northern, California'),
+('Pacific Northwest, USA', 'Pacific Northwest, USA'),
+('Hawaii, USA', 'Hawaii, USA'),
+('Northeast, USA', 'Northeast, USA'),
+('Southeast, USA', 'Southeast, USA'),
+('South America', 'South America'),
+('Europe', 'Europe'),
+('Africa', 'Africa'),
+('Southeast Asia', 'Southeast Asia'),
+('Asia', 'Asia');
```

Comment:
This is just adding the new responses into the table. Other is already a usable response, so not needed.